cardano-testnet/src/Test/Runtime.hs

```diff
@@ -58,14 +59,11 @@ data TestnetNode = TestnetNode
   , nodeProcessHandle :: IO.ProcessHandle
   }
 
+newtype TestnetNode = TestnetNode {unTestnetNode :: NodeRuntime}
```

Comment:
Let's get rid of `TestnetNode`. I don't see a clear benefits of wrapping `NodeRuntime` with it.